vertx-zipkin/src/main/java/io/vertx/tracing/zipkin/ZipkinTracerUtil.java

```diff
@@ -0,0 +1,95 @@
+package io.vertx.tracing.zipkin;
+
+import brave.Span;
+import brave.propagation.B3SingleFormat;
+import brave.propagation.TraceContext;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+
+public final class ZipkinTracerUtil {
+  public static final String ACTIVE_SPAN = "vertx.tracing.zipkin.active_span";
+  public static final String ACTIVE_CONTEXT = "vertx.tracing.zipkin.active_context";	  
+	
+  /**
+  * @return the current active {@link Span} otherwise {@code null}
+  */
+  public static Span getSpan() {
+    Context ctx = Vertx.currentContext();
+    if (ctx != null) {
+      return ctx.getLocal(ACTIVE_SPAN);
+    }
+    return null;
+  }
+
+  /**
+   * @return the current active {@link TraceContext} otherwise {@code null}
+   */
+  public static TraceContext getContext() {
```

Comment:
can you rename to currentTraceContext() ?

Reply:
done